test/unit/TopicCreateTransaction.js

```diff
@@ -0,0 +1,173 @@
+import { expect } from "chai";
+
+import {
+    TopicCreateTransaction,
+    TokenId,
+    CustomFixedFee,
+    PrivateKey,
+} from "../../src/index.js";
+
+describe("TopicCreateTransaction", function () {
+    describe("HIP-991: Permissionless revenue generating topics", function () {
+        it("should set correct the fee schedule key", function () {
+            const feeScheduleKey = PrivateKey.generateECDSA();
+            const topicCreateTransaction = new TopicCreateTransaction();
+
+            topicCreateTransaction.setFeeScheduleKey(feeScheduleKey);
+
+            expect(
+                topicCreateTransaction.getFeeScheduleKey().toString(),
+            ).to.eql(feeScheduleKey.toString());
+        });
+
+        it("should set fee exempt keys", function () {
+            const feeExemptKeys = [
+                PrivateKey.generateECDSA(),
+                PrivateKey.generateECDSA(),
+            ];
+
+            const topicCreateTransaction = new TopicCreateTransaction();
+
+            topicCreateTransaction.setFeeExemptKeys(feeExemptKeys);
+
+            feeExemptKeys.forEach((feeExemptKey, index) => {
+                expect(
+                    topicCreateTransaction.getFeeExemptKeys()[index].toString(),
+                ).to.eql(feeExemptKey.toString());
+            });
+        });
+
+        it("should add fee exempt key to empty list", function () {
+            const topicCreateTransaction = new TopicCreateTransaction();
+
+            const feeExemptKeyToBeAdded = PrivateKey.generateECDSA();
+
+            topicCreateTransaction.addFeeExemptKey(feeExemptKeyToBeAdded);
+
```

Comment:
Is there a reason we create the topicCreateTransaction and then add `setter` functionality. We could just do 

```
const topicCreateTransaction = new TopicCreateTransaction().addFeeExemptKey(...)```